corehq/motech/templates/motech/log_detail.html

```diff
@@ -50,6 +50,12 @@
               <pre>{{ log.request_error }}</pre>{% else %}&nbsp;
             {% endif %}</td>
           </tr>
+          <tr>
+            <th>{% trans "Response headers" %}:</th>
+            <td>{% if log.response_headers %}
+              <pre>{{ log.response_headers|pp_json }}</pre>{% else %}&nbsp;
```

Comment:
Might be weird for people looking at their old logs to see this be blank. Maybe add a note that this isn't always populated?